cleverhans/utils_cifar.py

```diff
@@ -0,0 +1,175 @@
+"""
```

Comment:
This and the SVHN reader might be better off in the directory for the example, since we aren't offering a standard library-wide way to read all different datasets.

Reply:
Fixed in cc8226d.